servicemigration/context.py

```diff
@@ -37,11 +57,9 @@ def __init__(self, filename=None):
 
     def commit(self, filename=None):
         """
-        Commits the service to the given filename. If filename is None,
-        writes to the file defined by sys.argv[2].
+        Commits the service to the given filename. Output precedence
+        is filename, MIGRATE_OUTPUTFILE, servicemigration endpoint.
```

Comment:
Again, per the comment above, probably needs a little more clarification if this the primary 'how to' doc we're going to give people.